### DISCUSSION TITLE
Is there a way for moviepy to help create text-based video editing?

### DISCUSSION THREAD

**Discussion by laurentHYZ** [Zulko/moviepy#1814] [Q&A]
Hi Zulko and everybody here, 
interested in text-based video editing ( the likes of Audiate, Techsmith etc). Are you working on something like that, or do you know if there is an existing open-source solution for that? 
thank you very much

**Comment by keikoro** [comment]
Please open a proper issue for this (using our issue templates) instead of using GitHub's Discussions feature, thank you.

**Reply by mgaitan** [reply]
What is this Discussion space for, if not for questions like these?

**Reply by keikoro** [reply]
@mgaitan Discussions shouldn't be set to active for this repository to begin with.

For the last couple of years, I've been the only person trying to maintain _some_ sense of order by categorising incoming issues and I am unwilling to double my efforts. There is no point to running two issue trackers in parallel without either clear guidelines regarding purpose/differentiation or the necessary personnel resources.

**Reply by mgaitan** [reply]
@keikoro sorry I did not mean to offend you, I apologize and thank you for your work that I also did for a while some time ago.  

I don't think "discussions" is *another* issue tracker.  Maybe it would be good to make that effort to discriminate what are mostly usage questions and refer them to discussions and leave the issue tracker to open actual bugs or feature requests.   

The advantage here is that the answers can be "marked as answer" and highlighted (alla stackoverflow style) we can build a useful, easy to use knowledge base for questions you and other are replying once an again. 

Also I guess people who are more of a library user can perhaps get involved in answering questions in discussion but will never pay attention if they are reported as "issues". 

what do you thing

**Reply by keikoro** [reply]
You didn't offend me, but if you've read the discussion about the future of the project, you'll be aware of where we're at, so your initial question seems rather naive to me.

Your argument for the Discussions board – while making perfect sense _in general_ – equally suggests to me you haven't fully grasped the situation. My point wasn't that GitHub don't know what they are doing; I'm well aware that Discussions aren't meant to be another issue tracker. But I don't have the capacity to _manage_ both an issue tracker and an additional forum. People using Discussions mainly to post issues is merely an _additional_ problem on top of that.   

If I had the necessary permissions to turn off Discussions, I'd have done so long ago. I redirect people to the tracker and lock discussion topics because it's the only way to keep everything in one place at a time we don't have the resources to look after multiple communication channels. A message board is nice in theory, but it's the wrong thing to prioritise at this point.

**Reply by mgaitan** [reply]
ok, you have my support if you think that's the best option. Obviously the work you have been doing is somewhat frustrating and although I intend to return to collaborate more systematically with the project, the truth is that it is the code itself that interests me more and not so much the support to the community of users. 

I don't have permissions either but we can summon @Zulko to explain the reasons and arguments. 

So @Zulko, would you agree to disable the discussion space to facilitate the management of issues and questions? Can you do that or give one of us the necessary permission to do so?  Thanks!

**Comment by mgaitan** [comment]
@laurentHYZ I  did an experimental project a few years ago that's based on remix a speech and produce the automatic edition (ie a [supercut](https://en.wikipedia.org/wiki/Supercut)) 

https://github.com/mgaitan/miau

here I wrote a blog post (in spanish, but I guess it's translatable) with some details 
http://mgaitan.github.io/posts/miau-cuando-el-gato-dice-la-verdad/

It's a bit clunky but it was fun to do. I would love to know that it works for you and you would be interested in improving it.
